### DISCUSSION TITLE
useFetch works on ssr mode is incorrect

### DISCUSSION THREAD

**Discussion by aliuq** [nuxt/nuxt#18000] [Questions]
use `useFetch` was got different result of **Terminal** and **Devtools** yestoday, but it got same result on before

```ts
const { data } = await useFetch(`${baseUrl}/search`, {
  method: 'POST',
  body: {
    q: formatKeyword,
    p: currentPage.value,
  },
})
console.log(1, data)
```

Terminal
```powershell
1 RefImpl {
  _shallow: false,
  dep: undefined,
  __v_isRef: true,
  _rawValue: { code: 1, data: { total: 52, rows: [Array], rows_latest: [Array] } },
  _value: { code: 1, data: { total: 52, rows: [Array], rows_latest: [Array] } }
}
```

Devtools
```
RefImpl {_shallow: false, dep: undefined, __v_isRef: true, _rawValue: null, _value: null}
```

Should I wrapper `useFetch` on server mode?

**Comment by andysay** [comment]
https://github.com/nuxt/framework/issues/2194 

useFetch make 2 request on 1 time in second return null

**Reply by lucassimines** [reply]
I'm having the same issue, when changing the page, then calling other `useFetch` using another `key` it returns `null`

**Comment by pi0** [comment]
Hi @liuqian1996. `data` key of `useFetch` is a [Ref](https://v3.vuejs.org/api/refs-api.html#ref) you should log it using `data.value`. What is `baseUrl` pointing to?

**Reply by aliuq** [reply]
this question reason maybe same as #2201

**Comment by rizkysyazuli** [comment]
Sorry to hijack the thread..

Hi @pi0, the docs and examples on data fetching does not mention anything about this. I'm currently trying to fetch data from a backend Laravel API from inside `<script setup>` with `ssr: false`. Like this:

```js
const runtimeConfig = useRuntimeConfig()
const { data } = await useFetch(`${runtimeConfig.public.apiURL}/foo`, {
  headers: {
    Authorization: `Bearer ${runtimeConfig.public.apiToken}`,
  },
})

console.log(data)
```

I assume `data` is usable at this point. But instead, what I see in the console is this:

```js
RefImpl {__v_isShallow: false, dep: undefined, __v_isRef: true, _rawValue: {…}, _value: Proxy(Object)}
```

Not sure what's different with my setup. I only got it working after I do it like this below. Clearly we still need to dig deeper for the actual data. The docs seriously needs some changes.

```js
console.log(data.value.data)
```
